src/brsTypes/components/RoAppInfo.ts

```diff
@@ -0,0 +1,140 @@
+import { BrsBoolean, BrsString, BrsValue, ValueKind } from "../BrsType";
+import { BrsComponent } from "./BrsComponent";
+import { BrsType } from "..";
+import { Callable, StdlibArgument } from "../Callable";
+import { Interpreter } from "../../interpreter";
+import * as PP from "../../preprocessor";
+
+export class RoAppInfo extends BrsComponent implements BrsValue {
+    readonly kind = ValueKind.Object;
+
+    private readonly manifest = PP.getManifestSync(process.cwd());
```

Comment:
Since the manifest doesn't change once the app is running, can we make this a `static` variable, so that we only ever read it once instead of each time we create a new `RoAppInfo`?

Reply:
Good call! Will take a look tomorrow.